### PR TITLE
Docs-md --> Docs

### PR DESCRIPTION
**Pull request by Quantalabs**
The new `docs-md` branch has updated docstrings in markdown, and uses pdoc, generating more readable documentation, as can be viewed [here](https://epispot.github.io/epispot/epispot/).

### CODE REVIEW

**Comment by quantum9Innovation** [comment]
Get rid of TravisCI on #docs after the merge

**Comment by quantum9Innovation** [comment]
Also increase code snippet readability after merge

File: .github/workflows/build.yml
Comment (quantum9Innovation) [inline .github/workflows/build.yml:23]: Remove whitespace

File: .github/workflows/coverage.yml
Comment (quantum9Innovation) [inline .github/workflows/coverage.yml:31]: Why is code coverage being changed on a docs branch?

File: .github/workflows/docs.yml
Comment (quantum9Innovation) [inline .github/workflows/docs.yml:22]: - [ ] Where is this being saved to?
 - [ ] Does this override current documentation?
- [ ] Has this workflow been tested?

File: sitemap.xml
Comment (quantum9Innovation) [inline sitemap.xml:1]: This sitemap will need to be edited for the new URL

**Review by quantum9Innovation — CHANGES_REQUESTED** [review]

File: .github/workflows/docs.yml
Comment (Quantalabs) [reply .github/workflows/docs.yml:22]: Saved to docs folder. Everything works and has been tested. This overrides any existing find to get up to date docs.

File: .github/workflows/coverage.yml
Comment (Quantalabs) [reply .github/workflows/coverage.yml:31]: This came from changes in the master branch.

**Comment by quantum9Innovation** [comment]
This PR is closed
**Do not merge a master-based branch into a nightly-based branch**

**Pull request closed without merging**
